sklearn/model_selection/_search.py

```diff
@@ -555,6 +559,22 @@ def classes_(self):
         self._check_is_fitted("classes_")
         return self.best_estimator_.classes_
 
+    def _generate_candidates(self):
+        """Generates lists of candidates to search as a coroutine
+
+        To be overridden by implementors.
+
+        It can iteratively generate a list of candidate parameter dicts, and is
```

Comment:
I think there is a grammatical mistake with this sentence (or I just don't understand it)